core/constants.py

```diff
@@ -48,3 +48,5 @@
 YEAR_CHOICES = []
 for year in range(2013, (datetime.now().year + 1)):
     YEAR_CHOICES.append((year, year))
+
+NEWLY_REPORTED = "Newly reported"
```

Comment:
You can add the other states as well

Reply:
that was to come in the next task, or i do it now?

Reply:
If there's a task for it it's fine..